### ISSUE TITLE
veraPDF does not invalidate PDF/A files with duplicate keys

### ISSUE THREAD

**Issue by petervwyatt** [veraPDF/veraPDF-library#1357]
veraPDF does not report that duplicate `OutputCondition` keys exist in the attached PDF/A-1 file.  

[SKM_C360i23062710180.pdf](https://github.com/veraPDF/veraPDF-library/files/11903911/SKM_C360i23062710180.pdf)

**Comment by bdoubrov** [comment]
The latest dev version reports all duplicated keys as warnings in the log:
```
WARNING: Dictionary/Stream contains duplicated key /OutputCondition(object key = 12 0 obj, offset = 10810)
```
This will be included in the next release as well.

**Comment by MaximPlusov** [comment]
Included into release 1.26
